plugins/CircuitViewer/io/CircuitLoader.cpp

```diff
@@ -688,7 +657,16 @@ ModelDescriptorPtr CircuitLoader::importFromFile(
     PropertyMap properties = getProperties();
     properties.merge(propertiesTmp);
     auto impl = Impl(_scene, properties);
-    return impl.importCircuit(filename, callback);
+    try
+    {
+        return impl.importCircuit(filename, callback);
+    }
+    catch (const std::exception& error)
+    {
+        BRAYNS_ERROR << "Failed to open " << filename << ": " << error.what()
```

Comment:
Why not let the exception throw? I think if you load from the web ui and this fails then you get no notification.

Reply:
OK, the try catch was already there, but from what you say it's better to remove it.